core-server/src/main/java/org/glassfish/jersey/server/ServerRuntime.java

```diff
@@ -669,11 +669,18 @@ public OutputStream getOutputStream(final int contentLength) throws IOException
 
             } catch (final Throwable ex) {
                 if (response.isCommitted()) {
-                    /**
-                     * We're done with processing here. There's nothing we can do about the exception so
-                     * let's just log it.
-                     */
-                    LOGGER.log(Level.SEVERE, LocalizationMessages.ERROR_WRITING_RESPONSE_ENTITY(), ex);
+                    if (ex.getCause() instanceof IOException) {
+                        skipFinally = true;
+                        LOGGER.log(Level.WARNING, "Response was sent, but there is IO issue", ex);
```

Comment:
There was an issue after the response was committed.
You don't know if the response was sent or not. (it could be sitting in a buffer somewhere locally, not even sent yet)
You just know you cannot change the response anymore.